file_dump.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Latte\Runtime as LR;
+/** DummyTemplateClass */
+final class DummyTemplateClass extends Latte\Runtime\Template
+{
+    public function main() : array
+    {
+        extract($this->params);
+        /** @var Symplify\PHPStanRules\Nette\Tests\Rules\NoLatteMissingMethodCallRule\Source\SomeTypeWithMethods $someType */
+        echo LR\Filters::escapeHtmlText($someType->getName());
+        echo "\n";
```

Comment:
@lulco Btw, it took me a while how to "fake" variables from Control/Presenter in rendered template in PHP. This is the way that works :)